src/util/utils.js

```diff
@@ -89,15 +90,15 @@ export const getFiatSymbol = (code: string) => {
 }
 
 export const logInfo = (msg: string) => {
-  console.log('%c ' + msg, 'background: grey; font-weight: bold; display: block;')
```

Comment:
None of these three functions are even used! We should just delete them altogether, since they are pointless (`console.warning` / `error` /  `info` already exist).

Reply:
fixed